custom_dc/quickstart.md

```diff
@@ -102,22 +110,47 @@ Note: If you are storing your source code in a public/open-source version contro
     </tr>
     <tr>
       <td><a href="https://github.com/datacommonsorg/website/blob/master/custom_dc/env.list"><code>custom_dc/env.list</code></a></td>
-      <td>Contains environment variables for the Data Commons services. For details of the variables, see the comments in the file.</td>
+      <td>Contains environment variables for locally run Data Commons data management and services containers. For details of the variables, see the comments in the file.</td>
     </tr>
   </tbody>
 </table>
 
-## Start the services {#start-services}
+## Load data
+
+In this step, we will add sample data that we have included as part of the download for you to load it into your custom instance. This data is from the Organisation for Economic Co-operation and Development (OECD): "per country data for annual average wages" and "gender wage gaps".
+
+To load the sample data:
 
 1. If you are running on Windows or Mac, start Docker Desktop and ensure that the Docker Engine is running.
-1. Open a terminal window, and from the root directory, run the following command:
+1. Open a terminal window, and from the root directory, run the following command to start the data management Docker container:
+
+```shell
+docker run \
+--env-file $PWD/custom_dc/env.list \
+-v $PWD/custom_dc/sample/:$PWD/custom_dc/sample  \
+gcr.io/datcom-ci/datacommons-data:stable
+```
+This does the following:
+
+- The first time you run it, downloads the latest stable Data Commons data image, `gcr.io/datcom-ci/datacommons-data:stable`, from the Google Cloud Artifact Registry, which may take a few minutes. Subsequent runs use the locally stored image.
+- Maps the input sample data to a Docker path.
+- Starts a Docker container.
+- Imports the data from the CSV files, resolves entities, and writes the data to a SQLite database file, `custom_dc/sample/datacommons/datacommons.db`.
+- Generates embeddings in `custom_dc/sample/datacommons/nl` and loads them. (To learn more about embeddings generation, see the [FAQ](faq.md#natural-language-processing).
+
+Once the container has executed all the functions in the scripts, it shuts down.
+
+## Start the services {#start-services}
+
+1. Open a new terminal window.
+1. From the root directory, run the following command to start the services Docker container:
 
 ```shell
 docker run -it \
 -p 8080:8080 \
 -e DEBUG=true \
 --env-file $PWD/custom_dc/env.list \
--v $PWD/custom_dc/sample:$PWD/custom_dc/sample  \
+-v $PWD/custom_dc/sample/:$PWD/custom_dc/sample  \
```

Comment:
Curious about why we added the trailing "/" here?

Reply:
Believe it or not, when I didn't include it, the loading would fail. Let me try it again and see if we need to make a point about it.

Reply:
Now I'm unable to reproduce this issue, so I don't know what it was about. Reverted this change.